transformer/app.js

```diff
@@ -225,11 +225,18 @@ export async function lambdaHandler(event, context) {
 
     let hubImportResponse;
     if (newFindingsToImport.length > 0) {
-      hubImportResponse = await hubClient.send(new BatchImportFindingsCommand({ Findings: newFindingsToImport }));
-      logger.debug('Security Hub import response', { hubImportResponse });
-
-      const dbResponse = await writeFindingsToDB(newFindingsToImport);
-      logger.debug('DB response', { dbResponse });
+      try {
+        hubImportResponse = await hubClient.send(new BatchImportFindingsCommand({ Findings: newFindingsToImport }));
+        logger.debug('Security Hub import response', { hubImportResponse });
+      } catch (e) {
+        logger.debug(`Error while importing findings ${e}`);
```

Comment:
Please use `logger.error()`

Reply:
Let's not use string interpolation here. Use the same pattern as other `logger` calls, e.g. `logger.error('Error while importing findings', { e })`.

Reply:
fixed